content/hardware/04.pro/boards/portenta-x8/datasheet/datasheet.md

```diff
@@ -435,7 +397,9 @@ Sample sketches for the Arduino® Portenta X8 can be found either in the “Exam
 Now that you have gone through the basics of what you can do with the board you can explore the endless possibilities it provides by checking exciting projects on ProjectHub **[5]**, the Arduino® Library Reference **[6]** and the online store **[7]** where you will be able to complement your board with sensors, actuators and more.
 
 ### Board Recovery
-All Arduino boards have a built-in bootloader which allows flashing the board via USB. In case a sketch locks up the processor and the board is not reachable anymore via USB it is possible to enter bootloader mode by double-tapping the reset button right after power up.
+All Arduino boards have a built-in bootloader which allows flashing the board via USB. In case a sketch locks up the processor and the board is not reachable anymore via USB it is possible to enter bootloader mode by configuring DIP switches. 
+
+***Note: A compatible carrier board with DIP switches (e.g. Portenta Max Carrier or Portenta Breakout) is required to enable bootloader mode. Bootloader cannot be reached with the Portenta X8 alone.***
```

Comment:
```suggestion
**Note: A compatible carrier board with DIP switches (e.g. Portenta Max Carrier or Portenta Breakout) is required to enable bootloader mode. It cannot be enabled with the Portenta X8 alone.**
```

Reply:
Should we use three asterisks or two for notes? @marqdevx

Reply:
@aliphys Three usually, but it won't be rendered as a note block inside the datasheets, only in the tutorials.